doc/content/integrations/pubsub/mqtt-client/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Configure MQTT (Client)"
+description: ""
+weight: 1
+---
+
+This section explains how to configure a PubSub Integration using the built in MQTT client and an external MQTT server.
+
+<!--more-->
+
+Creating a PubSub integration with the built in MQTT Client requires you to have an MQTT server running with an available endpoint. [Mosquitto](https://mosquitto.org/) is a popular open source MQTT Server you may use, but {{% tts %}} MQTT client should work with any MQTT server.
+
+In your application select the **Pub-subs** submenu from the **Integrations** side menu. Clicking on the **+ Add pub-sub** button will open the PubSub creation screen.
```

Comment:
"PubSubs" right? If it says "Pub-subs" and "pub-sub" we have an inconsistency we should fix there

Reply:
true. It's most commonly written **Pub/Sub** in the wild - should we go with that?

Reply:
Yeah had some problems finding a conventional way to write it. Maybe Pub/Subs is indeed better. I'll make a PR to change this.